app/views/proposal_mailer/proposal_submitted_approvers_notification.html.erb

```diff
@@ -1,12 +1,12 @@
-<h1 class="styled-heading"><%= "#{@short_name}_#{@version} Submitted" %></h1>
+<h1 class="styled-heading"><%= "#{@short_name}_#{@version} Submitted in #{Rails.configuration.cmr_env.upcase}" %></h1>
```

Comment:
Have you checking with Alicia on this? I was sort of expecting there to be no change to the e-mails sent from prod

Reply:
Checked, now updated